receiver/hostmetricsreceiver/internal/scraper/pagingscraper/documentation.md

```diff
@@ -11,6 +11,7 @@ These are the metrics available for this scraper.
 | **system.paging.faults** | The number of page faults. | {faults} | Sum(Int) | <ul> <li>type</li> </ul> |
 | **system.paging.operations** | The number of paging operations. | {operations} | Sum(Int) | <ul> <li>direction</li> <li>type</li> </ul> |
 | **system.paging.usage** | Swap (unix) or pagefile (windows) usage. | By | Sum(Int) | <ul> <li>device</li> <li>state</li> </ul> |
+| system.paging.utilization | Percentage of Swap (unix) or pagefile (windows) utilization. | 1 | Gauge(Double) | <ul> <li>device</li> <li>state</li> </ul> |
```

Comment:
According to
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming utilization metrics has to be a fraction of 1, not percentage.